jobs/payment-jobs/tasks/routing_slip_task.py

```diff
@@ -333,6 +348,98 @@ def _get_routing_slip_by_status(cls, status: RoutingSlipStatus) -> list[RoutingS
             .all()
         )
 
+    @classmethod
+    def _get_applied_invoices_amount(cls, routing_slip: RoutingSlipModel) -> float:
+        """Get total amount of applied invoices in SBC-PAY for routing slip."""
+        total_applied = (
+            db.session.query(func.sum(InvoiceModel.total))
+            .filter(
+                InvoiceModel.routing_slip == routing_slip.number,
+                InvoiceModel.invoice_status_code == InvoiceStatus.PAID.value
+            )
+            .scalar()
+        ) or 0.0
+
+        return float(total_applied)
```

Comment:
probably better using `Decimal(str(total_applied))`
or cast as numeric (Decimal equivalent)


If you use decimal math you wont have to worry about weird float math issues

Sec let me find examples

<img width="1180" height="633" alt="image" src="https://github.com/user-attachments/assets/779c9982-116b-41f4-b3da-2a8ff067ec4a" />

<img width="939" height="821" alt="image" src="https://github.com/user-attachments/assets/371d0e03-ffbd-490d-a2cd-ce6a2465dcce" />